.env

```diff
@@ -0,0 +1,2 @@
+# Do not put this file to version controlled in the real project
+SECRET=From production with love!
```

Comment:
Do you think we should guide dev to create these files in README after checked out the template instead of keeping them here in git?

Reply:
@luongvo I think we can keep them in the template repo to reduce the effort when the devs(not Flutter devs) want to check out the repo and don't have to ask others for getting environment variables. How do you think? 🙏

Reply:
empty file won't have any value, and you shouldn't expose the `KEY` in production neither, because that's 50% of the secret already. Please ref to https://github.com/nimblehq/compass to learn how to manage .env file

Reply:
@sleepylee @luongvo  ah I see. Let me update, and add the guide to README as well

Reply:
@sleepylee @luongvo Updated 🙏 I removed `.env` and `.env.staging`, and updated README 🚀